src/app.js

```diff
@@ -198,7 +194,7 @@ class SAFEApp extends EventEmitter {
         const serviceInfo = await getServiceInfo(lookupName, serviceName);
         if (serviceInfo.buf.length === 0) {
           const error = new Error();
-          error.code = ERR_NO_SUCH_ENTRY;
+          error.code = errConst.ERR_NO_SUCH_ENTRY;
```

Comment:
I think we should be consistent with whats in the consts. And so make this a `.code` over there.